internal/service/elasticache/replication_group.go

```diff
@@ -490,7 +499,12 @@ func resourceReplicationGroupCreate(ctx context.Context, d *schema.ResourceData,
 		input.PreferredMaintenanceWindow = aws.String(v.(string))
 	}
 
-	if v, ok := d.GetOk("multi_az_enabled"); ok {
+	// We want to explictly set MultiAZEnabled even if it is `false` in the configuration.
+	// This addresses the error InvalidParameterCombination
+	// "Each Node Group needs to have at least one replica for Multi-AZ enabled Replication Group"
+	// when creating a replication group with 1 shard/node-group and 0 replicas with Multi AZ disabled,
+	// using the snapshot of either a multi-AZ enabled replication group or of a serverless cache.
+	if v, ok := d.GetOkExists("multi_az_enabled"); ok {
```

Comment:
This seems to have an unintended effect of always setting `input.MultiAZEnabled`, even if the `multi_az_enabled` attribute is not specified in the resource or is set as `null`.
What's the best way to check if an attribute exists in the configuration, regardless of whether it's value is zero or not.
`GetOk` function has the downside that will return the second value as `false` if the first one is a "type's zero"